lesson3/bigForm1/polymer/README.md

```diff
@@ -3,5 +3,7 @@ To install and run the Polymer checkout form:
 1. Install [npm](https://www.npmjs.com/), [bower](http://bower.io/) and [gulp](http://gulpjs.com/) if you haven't already
 2. Run `npm install`
 3. Run `bower install`
-4. Host over `localhost` and open `dest/index.html` in your browser
-5. As you make your edits to the files in `src/`, use `gulp watch` to watch for changes and rebuild to `dest/`.
\ No newline at end of file
+4. Run `gulp serve`
+5. Browser should open to http://localhost:3000/ -- navigate to /src (changes in src automatically refresh in browser)
```

Comment:
I thought I would run this by you:

I've tried serving files from `src` and `dest`, but then `bower_components` and `images` can no longer be accessed . I tried to get around this with the browserSync `routes` option, but didn't have any luck. Unless you know a way to do this with browserSync, then a redirect is a great solution.

I really think serving files from `src` is a great option for development. I was thinking two redirect files, one for each folder.
- `gulp serve` would default to `dest`
- `gulp serve --src`

I can also just do the one redirect, run the default task and serve `dest`

Let me know :-)